### ISSUE TITLE
Mythic Classes + Isekai Races

### ISSUE THREAD

**Issue by Nerin3** [JohN100x1/IsekaiMod#180]
I was just thinking, but how about giving some flavour for certain mythic paths if you choose a specific race (For simplicity I call everything Isekai <Race Here>)
For Example:
Isekai Dhampir + Lich - Can pick lich powers as normal feats
+ some flavour if possible: Lets you keep your body, and defy Zacharius, but still get one additional level by the virtue of being a "superior" Undead being.
Isekai Tiefling + Demon/Devil - Gives you a permanent Aspect of Succubus as well as upgrades the charm to refresh every 1d6 turns.
Isekai Human + Legend - You can pick mythic feats and abilities as normal feats (or you don't lose your mythic feats that you've chosen as you progress the legend).
Isekai Elf/Gnome + Azata - Can pick azata's superpowers as normal feats
Isekai Kitsune/Gnome + Trickster - Can pick mythic tricks as normal feats
Isekai Oread + Aeon - Can pick Aeon gaze as normal feats
Isekai Aasimar + Angel - Can pick Angel Halo and Sword as normal feats

**Comment by Brody1018** [comment]
I would definitely try it out if the feature was made available. I just don't know that I would like being pigeonholed into a certain race in order to fully appreciate the mod.

**Comment by bundillion** [comment]
> pigeonholed into a certain race in order to fully appreciate the mod.

I disagree that this idea is limiting player options so much as providing additional benefits to the special options the mod gives. Builds that are internally complimentary, either through mechanics or flavor or both, deserve build-specific treatment. It enhances replayability for those wanting to explore every avenue, without feeling so generic that it wouldn't matter at all what class and race you play as. Is it maybe a bit obtuse for a class mod to have that much depth? I think that's a matter of opinion (and also probably yes).
